### DISCUSSION TITLE
Has terrain exaggeration been removed？eg：<terrain  vertical_scale="5"/>

### DISCUSSION THREAD

**Discussion by GaoboLi** [pelicanmapping/osgearth#2118] [Support Q&A]
I want to exaggerate the terrain in the Rex engine, but vertical_ Scale no longer works.

Another problem is how to add a shared layer in the latest version.

**Comment by gwaldron** [comment]
The rex terrain engine does not have a vertical scaling option.

You can specify a shared layer by the property `shared="true"`. You can then set the sampler and matrix uniforms like so:
```xml
<TMSImage name="my layer" shared="true">
    <url>...</url>
    <shared_sampler>MY_SAMPLER</shared_sampler>
    <shared_matrix>MY_MATRIX</shared_matrix>
</TMSImage>
```
